weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -333,35 +337,39 @@ const MainPeekingLayout: FC = () => {
           flex: '1 1 40%',
           overflow: 'hidden',
           display: 'flex',
+          // This transition is from the mui drawer component, to keep the main content animation in similar
+          transition: !isDrawerOpen
+            ? 'margin 225ms cubic-bezier(0, 0, 0.2, 1) 0ms'
+            : 'none',
+          marginRight:
+            !isDrawerOpen || !isFlexRow
+              ? 0
+              : // this is a hack to make the drawer width percentage work with the siderbar of 56px
+                `${drawerWidthPct / (1 - 56 / windowSize.width)}%`,
```

Comment:
This `56` and the `60` for the header feel like magic numbers that could change. Can we pull those into constants?

Note: it is sort of odd that this component "knows" about the space it is mounted in. Fine for now, but just calling out that it is an odd coupling between sibling/cousin components.